src/js/components/StatusBar.js

```diff
@@ -11,22 +11,72 @@ class StatusBar extends React.Component {
     if (max === 0) {
       return null;
     }
+    let indexesLessThanThreshold = [];
+    // Size of largest portion to give to smallest portion. We have a max of 5
+    // status types so we want to keep this growth ratio relatively small.
+    let growRatio = 0.07;
+    // Grow anything below this threshold by stealing from the biggest portion
+    let threshold = 7;
+
+    let barSizes = data.map(function (status, index) {
+      let {value} = status;
+
+      let relativeSize = value / max * 100;
+
+      if (relativeSize > 0 && relativeSize < threshold) {
+        indexesLessThanThreshold.push(index);
+      }
+
+      return relativeSize;
+    });
+
+    function stealPortion() {
```

Comment:
what do you think about having `growRatio` and `threshold` as consts on top of the file, then having `stealPortion` taking in `barSizes` and `indexesLessThanThreshold` as arguments, then we can put this function at the top of the file too